src/System.Net.Http/src/netcore50/System/Net/HttpClientHandler.cs

```diff
@@ -642,46 +669,6 @@ private void CheckDisposedOrStarted()
             }
         }
 
-        private RTPasswordCredential RTPasswordCredentialFromNetworkCredential(NetworkCredential creds)
-        {
-            // RTPasswordCredential doesn't allow assigning string.Empty values, but those are the default values.
-            RTPasswordCredential rtCreds = new RTPasswordCredential();
-            if (!string.IsNullOrEmpty(creds.UserName))
-            {
-                if (!string.IsNullOrEmpty(creds.Domain))
-                {
-                    rtCreds.UserName = creds.Domain + "\\" + creds.UserName;
-                }
-                else
-                {
-                    rtCreds.UserName = creds.UserName;
-                }
-            }
-            if (!string.IsNullOrEmpty(creds.Password))
-            {
-                rtCreds.Password = creds.Password;
-            }
-
-            return rtCreds;
-        }
-        
-        private void SetProxyCredential(IWebProxy proxy)
-        {
-            // We don't support changing the proxy settings in the NETNative version of HttpClient since it's layered on
-            // WinRT HttpClient. But we do support passing in explicit proxy credentials, if specified, which we can
-            // get from the specified or default proxy.
```

Comment:
Sorry, I forgot to remove this from the review: I did see that the code handling explicit proxy credentials has been moved (not removed as I've originally thought).